library/tomee

```diff
@@ -1,49 +1,49 @@
 Maintainers: Alex Soto <asotobu@gmail.com> (@lordofthejars), Otavio Santana <otaviojava@apache.org> (@otaviojava), Jonathan Gallimore <jgallimore@apache.org> (@jgallimore), Rod Jenkins <jenkir14@nationwide.com> (@scriptmonkey)
 GitRepo: https://github.com/tomitribe/docker-tomee.git
-GitCommit: 1de01bc405415fffc3c1c6d1f95ad0af487417c4
+GitCommit: 98ce59346d56f8a754f70f08a37276b93eb15c1a
 Architectures: amd64
 
-Tags: 8-jre-7.0.8-plume, 7.0.8-plume
+Tags: 8-jre-7.0.9-plume, 7.0.9-plume
 Directory: TomEE-7.0/jre8/plume
 
-Tags: 8-jre-7.0.8-plus, 7.0.8-plus
+Tags: 8-jre-7.0.9-plus, 7.0.9-plus
 Directory: TomEE-7.0/jre8/plus
 
-Tags: 8-jre-7.0.8-webprofile, 7.0
+Tags: 8-jre-7.0.9-webprofile, 7.0
 Directory: TomEE-7.0/jre8/webprofile
 
-Tags: 8-jre-7.1.3-plume, 7.1.3-plume
+Tags: 8-jre-7.1.4-plume, 7.1.4-plume
 Directory: TomEE-7.1/jre8/plume
 
-Tags: 8-jre-7.1.3-plus, 7.1.3-plus
+Tags: 8-jre-7.1.4-plus, 7.1.4-plus
 Directory: TomEE-7.1/jre8/plus
 
-Tags: 8-jre-7.1.3-microprofile, 7.1.3-microprofile
+Tags: 8-jre-7.1.4-microprofile, 7.1.4-microprofile
 Directory: TomEE-7.1/jre8/microprofile
 
-Tags: 8-jre-7.1.3-webprofile, 7.1.3-webprofile, 7, 7.1
+Tags: 8-jre-7.1.4-webprofile, 7.1.4-webprofile, 7, 7.1
 Directory: TomEE-7.1/jre8/webprofile
 
-Tags: 8-jre-8.0.4-plume
+Tags: 8-jre-8.0.5-plume
 Directory: TomEE-8.0/jre8/plume
 
-Tags: 8-jre-8.0.4-plus
+Tags: 8-jre-8.0.5-plus
 Directory: TomEE-8.0/jre8/plus
 
-Tags: 8-jre-8.0.4-microprofile
+Tags: 8-jre-8.0.5-microprofile
 Directory: TomEE-8.0/jre8/microprofile
 
-Tags: 8-jre-8.0.3-webprofile
+Tags: 8-jre-8.0.5-webprofile
 Directory: TomEE-8.0/jre8/webprofile
 
-Tags: 11-jre-8.0.4-plume, 8.0.4-plume
+Tags: 11-jre-8.0.5-plume, 8.0.5-plume
 Directory: TomEE-8.0/jre11/plume
 
-Tags: 11-jre-8.0.4-plus, 8.0.4-plus
+Tags: 11-jre-8.0.5-plus, 8.0.5-plus
 Directory: TomEE-8.0/jre11/plus
 
-Tags: 11-jre-8.0.4-microprofile, 8.0.4-microprofile
+Tags: 11-jre-8.0.5-microprofile, 8.0.5-microprofile
 Directory: TomEE-8.0/jre11/microprofile
 
-Tags: 11-jre-8.0.3-webprofile, 8.0.2-webprofile, latest, 8
+Tags: 11-jre-8.0.5-webprofile, 8.0.2-webprofile, latest, 8
```

Comment:
:confused: `8.0.2-webprofile` for `11-jre-8.0.5-webprofile`? (Seems it was also a bad tag for `11-jre-8.0.3-webprofile`)

Reply:
Well shoot, I will fix.